AirCasting/Dashboard/DashboardView.swift

```diff
@@ -7,26 +7,37 @@
 
 import CoreData
 import SwiftUI
+import AirCastingStyling
+import Combine
 
 struct DashboardView: View {
     #warning("This hook fires too often - on any stream measurement added/changed. Should only fire when list changes.")
     @StateObject var coreDataHook: CoreDataHook
     @FetchRequest<SensorThreshold>(sortDescriptors: [.init(key: "sensorName", ascending: true)]) var thresholds
     @EnvironmentObject var selectedSection: SelectSection
     @EnvironmentObject var averaging: AveragingService
+    @State var isRefreshing: Bool = false
     
-    let measurementStreamStorage: MeasurementStreamStorage
-    let sessionStoppableFactory: SessionStoppableFactory
+    private let measurementStreamStorage: MeasurementStreamStorage
+    private let sessionStoppableFactory: SessionStoppableFactory
+    private let sessionSynchronizer: SessionSynchronizer
+    
+    private let dashboardCoordinateSpaceName = "dashboardCoordinateSpace"
```

Comment:
Do we want to move it into Strings?

Reply:
We only put user-facing strings there (so we have them all in a single place), this one is not user facing, just a key

Reply:
Got it - thanks, so no doubts then 😄